pages/dkp/kaptain/2.0.0/install/deploy-kaptain/index.md

```diff
@@ -150,6 +150,26 @@ NAME                      AGE     READY   STATUS
 kaptain-1                 3m40s   True    Release reconciliation succeeded
 ```
 
+## Log in to Kaptain using the management cluster's Dex instance
+
+Discover the Kaptain endpoint:
+
+- If you are running Kaptain _on-premises_:
+
+```bash
+kubectl get svc kaptain-ingress --namespace kaptain-ingress -o jsonpath="{.status.loadBalancer.ingress[*].ip}"
+```
+<!-- TODO: can we get example output? -->
+
+- Or if you are running Kaptain on _AWS_:
+
+```bash
+kubectl get svc kaptain-ingress --namespace kaptain-ingress -o jsonpath="{.status.loadBalancer.ingress[*].hostname}"
+```
+<!-- TODO: can we get example output? -->
+
+When calling up `https://<Kaptain endpoint>`, you will see the login page of the management cluster's Dex instance. After entering your credentials, you will be redirected to Kaptain's Kubeflow dashboard.
```

Comment:
should we put the step to get the username/pw again? [Get the login credentials from Konvoy to authenticate:](https://docs.d2iq.com/dkp/kaptain/1.3.0/install/konvoy-dkp/#:~:text=Get%20the%20login%20credentials%20from%20Konvoy%20to%20authenticate%3A)

Reply:
definitely, good idea!

Reply:
This may be different for those with hosted environs, but I have noticed how on browsers, it does NOT like the Kaptain login security business.
You have to use the [thisisunsafe](https://stackoverflow.com/questions/35274659/when-you-use-badidea-or-thisisunsafe-to-bypass-a-chrome-certificate-hsts-err) workaround.
I guess that's more of a comment than a suggestion for the docs - but we might want to say something that it's a trusted source?